index.js

```diff
@@ -6,6 +6,18 @@ const path = require('path');
 
 const parseIdentifier = require('./parse-identifier');
 
+function isEmberApp(pkg) {
+  if (pkg.devDependencies && pkg.devDependencies['ember-source']) {
+    return true;
+  }
+
+  if (pkg.dependencies && pkg.dependencies['ember-source']) {
+    return true;
+  }
```

Comment:
Hmm, this will likely also be true for things other than an app. One example is when we get V2 Addons.

Reply:
Hmm that's a good point.  That said, once we're in embroider mode the need for this basically goes away since then relative imports will work

Reply:
Maybe that's an argument for making a ("temporaryish") wrapper utility and leave this one conceptually simpler